src/data/icons/svg/unobtanium.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="600.000000pt" height="600.000000pt" viewBox="0 0 600.000000 600.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.15, written by Peter Selinger 2001-2017
+</metadata>
+<g transform="translate(0.000000,600.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M1267 5750 c-55 -14 -110 -45 -148 -82 -41 -42 -81 -128 -87 -190
+l-5 -48 91 0 90 0 7 33 c11 49 47 107 77 123 15 8 46 14 69 14 33 0 47 -6 75
+-34 31 -31 34 -40 34 -90 0 -95 -35 -144 -193 -278 -196 -165 -255 -246 -265
+-364 l-5 -64 327 0 326 0 0 85 0 85 -195 0 c-113 0 -195 4 -195 9 0 5 35 40
+78 77 134 118 250 241 279 296 25 47 28 63 28 148 0 78 -4 102 -22 135 -49 92
+-138 144 -256 150 -40 3 -89 0 -110 -5z"/>
+<path d="M1972 5741 c-96 -33 -161 -118 -199 -261 -24 -93 -25 -369 0 -459 35
+-130 83 -202 161 -244 34 -18 57 -22 141 -22 91 0 104 2 146 27 127 75 174
+203 173 473 0 173 -12 244 -59 341 -66 136 -216 196 -363 145z m168 -169 c44
+-44 62 -118 67 -277 9 -231 -23 -354 -99 -383 -78 -30 -138 36 -160 175 -15
+95 -5 334 17 402 22 71 62 111 110 111 27 0 44 -8 65 -28z"/>
+<path d="M596 5709 c-13 -65 -116 -129 -208 -129 l-28 0 0 -79 0 -79 68 5 c37
+3 84 9 105 13 l37 8 0 -339 0 -339 95 0 95 0 0 485 0 485 -79 0 -78 0 -7 -31z"/>
+<path d="M480 2798 c0 -670 4 -1169 10 -1237 21 -233 62 -403 137 -561 58
+-121 90 -167 177 -258 158 -164 356 -263 651 -324 101 -21 136 -23 420 -23
+282 0 321 2 426 23 436 85 718 304 857 667 34 89 71 250 94 407 21 149 22 184
+25 1291 l4 1137 -335 0 -334 0 -5 -1127 c-5 -1194 -4 -1169 -52 -1353 -9 -36
+-35 -102 -57 -147 -103 -212 -276 -296 -613 -296 -413 -1 -593 118 -691 458
+-14 49 -31 126 -36 170 -7 49 -12 503 -15 1188 l-4 1107 -330 0 -329 0 0
+-1122z"/>
+<path d="M4406 3015 c-124 -21 -244 -70 -358 -146 l-58 -38 0 69 0 70 -320 0
+-320 0 0 -1265 0 -1265 320 0 320 0 0 865 0 865 23 26 c52 62 201 175 283 216
+86 42 90 43 203 46 111 3 118 2 171 -25 61 -32 95 -76 126 -163 18 -52 19 -96
+22 -943 l3 -887 320 0 320 0 -3 933 c-3 922 -4 933 -25 1023 -90 368 -312 570
+-683 620 -128 17 -239 16 -344 -1z"/>
+</g>

```

Comment:
SVG code must be simplified like other icons so we can generate the component assets